src/Login.js

```diff
@@ -88,12 +85,33 @@ class Login extends Component {
           {this.state.error && (
             <Alert bsStyle="danger">{this.state.error}</Alert>
           )}
-          <FieldGroup
-            inputType="button"
-            label=""
-            buttonText="Submit"
-            onClick={this.onSubmit}
-          />
+
+          {this.state.enter_edit && (
+            <div>
+              <FieldGroup
+                onClick={this.props.history.push('/', { isEditing: true })}
+              />
+            </div>
+          )}
+
+          {this.state.logged_in && (
+            <div>
+              <FieldGroup
+                inputType="button"
+                buttonText="Enter Edit Mode"
+                onClick={this.returnToMain}
```

Comment:
When the user logs in, it might make more sense to automatically redirect them instead of having them click a button to do it (especially since there's nothing left for them to do on the login page)